packages/invoices/src/OutboundShipment/ListView/Toolbar.tsx

```diff
@@ -56,23 +55,14 @@ export const Toolbar: FC<{
         display: 'flex',
       }}
     >
-      <InputWithLabelRow
-        label={t('label.search')}
-        labelWidth={null}
-        Input={
-          <BasicTextInput
-            value={filterString}
-            placeholder={t('placeholder.enter-a-customers-name')}
-            onChange={e =>
-              filter.onChangeStringFilterRule(
-                'otherPartyName',
-                'like',
-                e.target.value
-              )
-            }
-          />
-        }
+      <SearchBar
+        placeholder="Search by comment..."
```

Comment:
the api doesn't support filtering by other party name - have changed to comment for now. Didn't add a translation (and so it begins?) because no one wants to search by comment, normally, right!

Reply:
Fixed the inbound not creating - the search issues I think are resolved with backend fix